test/document_test.rb

```diff
@@ -443,6 +443,17 @@ def setup
     assert_equal [], second_doc.foo
   end
 
+  test 'attachments work properly' do
```

Comment:
`attach_file will add a standalone attachment to document`
`attach standalone document`
`add attachment to document`
`attachment can be added to document`
Not trying to be picky, just something more descriptive than work properly.